scheds/rust/scx_p2dq/src/lib.rs

```diff
@@ -232,6 +292,18 @@ pub struct SchedulerOpts {
     #[clap(long, default_value_t = false, action = clap::ArgAction::Set)]
     pub virt_llc_enabled: bool,
 
+    /// Enable hardware-specific optimizations automatically
+    #[clap(long, default_value_t = true, action = clap::ArgAction::Set)]
+    pub hw_auto_optimize: bool,
+
+    /// Force single-LLC fast path optimizations
+    #[clap(long, action = clap::ArgAction::SetTrue)]
+    pub single_llc_fast_path: bool,
+
+    /// Enable ARM64 Neoverse-V2 optimizations (auto-detected by default)
+    #[clap(long, action = clap::ArgAction::SetTrue)]
+    pub arm64_neoverse_v2: bool,
```

Comment:
This is unused?

Reply:
whoops, I think I was going to make it optional at first and left it in